ui/screens.py

```diff
@@ -1323,8 +1323,12 @@ def setPools(self, pools):
             return
         self.pools = pools
         tNow = int(time.time())
-        # only save pools updated within the last day
-        self.pools = [p for p in pools if tNow - p["LastUpdated"] < 86400 and self.scorePool(p) > 95]
+        # Only save pools updated within the last day, but allow bad pools for
+        # testing.
+        # TODO: Have 3 tinydecred network constants retreivable through cfg
+        #   instead of checking the network config's Name attribute.
+        if cfg.net.Name == "mainnet":
+            self.pools = [p for p in pools if tNow - p["LastUpdated"] < 86400 and self.scorePool(p) > 95]
```

Comment:
Pools have a simple filter for inactivity or really poor performance. Unfortunately, 2 of the 3 listed testnet VSPs do not currently pass. Disabling filtering for testnet for now.